### DISCUSSION TITLE
How to override keyboard arrows (remap to IJKL)

### DISCUSSION THREAD

**Discussion by drschwabe** [wezterm/wezterm#5896] [General]
Would like to override the default up/down/left/right to ALT+IJKL  (similar hotkeys to 'vim' style ie- so i dont have to move my right hand to move the caret) 

I have a confirmed working wezterm.lua config but I cannot seem to figure out how to get these desired keybindings.   Any tips appreciated.

**Comment by nekowinston** [comment]
You could use [wezterm.action.SendKey](https://wezfurlong.org/wezterm/config/lua/keyassignment/SendKey.html) to emulate pressing the arrow keys.

```lua
local wezterm = require("wezterm")
local act = wezterm.action
local config = wezterm.config_builder()

config.keys = {
  {
    key = "h",
    mods = "ALT",
    action = act.SendKey({
      key = "LeftArrow",
    }),
  },
  {
    key = "j",
    mods = "ALT",
    action = act.SendKey({
      key = "DownArrow",
    }),
  },
  {
    key = "k",
    mods = "ALT",
    action = act.SendKey({
      key = "UpArrow",
    }),
  },
  {
    key = "l",
    mods = "ALT",
    action = act.SendKey({
      key = "RightArrow",
    }),
  },
}

return config
```

**Comment by drschwabe** [comment]
Awesome, thanks
